### ISSUE TITLE
Add an option to remove the magnifying glass indicator from the toolbar icon

### ISSUE THREAD

**Issue by ruohola** [cavi-au/Consent-O-Matic#286]
I really like the 'hide' display option that Consent-O-Matic has, where all the magic just happens in the background.

It would be nice to have an option to additionally also not make this magnifying glass indicator pop up to the toolbar icon whenever I load a web page on Mac:

<img width="128" alt="image" src="https://github.com/cavi-au/Consent-O-Matic/assets/33625218/68fc2aeb-314d-4f87-bc3e-fcfa63da51dc">

Also, similarly on an iPhone, this blue dot appears whenever I load a page, even when the page doesn't have a cookie banner:

![IMG_4411](https://github.com/cavi-au/Consent-O-Matic/assets/33625218/0dc188e9-c9bf-4676-865b-6a6de98fb6ad)

It would be nice to have an option to disable both of these. Maybe even with an additional option that the indicator would only appear if a cookie banner was blocked in the background.

**Comment by MidasN** [comment]
I don't think we have the capacity ourselves for this at the moment, but we'd be happy to receive a PR!
